torch_tvm/register.cpp

```diff
@@ -72,5 +75,33 @@ PYBIND11_MODULE(_torch_tvm, m) {
 
   m.def("disable", []() { fusion_enabled = false; });
 
+  m.def("push_relay_expr", [](std::shared_ptr<Graph> g,
```

Comment:
since we don't expose the `push_relay_expr` as a public API, should we make it as `_push_relay_expr`?